src/MembershipsSDKAdapter.test.js

```diff
@@ -0,0 +1,58 @@
+import {isObservable} from 'rxjs';
+import {DestinationType} from '@webex/component-adapter-interfaces';
+
+import MembershipsSDKAdapter from './MembershipsSDKAdapter';
+import createMockSDK from './__mocks__/sdk';
+
+describe('Memberships SDK Adapter', () => {
+  let membershipSDKAdapter, mockSDK, meetingID;
+
+  beforeEach(() => {
+    mockSDK = createMockSDK();
+    membershipSDKAdapter = new MembershipsSDKAdapter(mockSDK);
+    meetingID = 'meetingID';
+  });
+
+  describe('getMembersFromDestination()', () => {
+    test('returns an observable', () => {
+      expect(isObservable(membershipSDKAdapter.getMembersFromDestination())).toBeTruthy();
+    });
+
+    describe('meeting destination type', () => {
```

Comment:
Not sure the nested `describe` are necessary? it makes the test a bit confusing since it's not just about the meeting destination or the meeting type

Reply:
Since we will have different tests for the different destination types, it is easier to group the tests into the blocks for each situation. I will change the describe block so that the test reads a bit better:
`Memberships SDK Adapter getMembersFromDestination() when destination type is ROOM throws an error on subscription`